### PR TITLE
Corrected homepage style

### LINKED ISSUE #8: Fix style issues on the home screen
As shown below, when the page is resized, some elements are breaking.
Screenshot_20201119_202810

### PR DESCRIPTION
**Pull request by claytonrv**
Corrected the homepage style that was broken, as shown on the issue description.

### CODE REVIEW

**Comment by claytonrv** [comment]
Final result 
Screenshot_20201202_220337

File: src/pages/Landing.tsx [hunk 1]
```diff
@@ -15,13 +15,23 @@ function Landing() {
...
+                            <div>
+                                <div className="text_grid">
+                                    <h1>Our lab to try tech</h1>
+                                    <div className="green-dot"></div>
+                                </div>
+                            </div>
+                            <p>We're a group of friends trying to do cool things.</p>
+                        </div>
+                        <div className="grid_column">
+                            <img src={Developer} alt="Man front a laptop" className="home-developer" />
```
Comment (lucasmauro) [inline src/pages/Landing.tsx]: Small stuff, but can we change this to `man in front of a laptop`?

File: src/pages/Developers.tsx [hunk 2]
```diff
@@ -23,6 +23,18 @@ export default function Developers() {
     useEffect(() => {
         getDevelopers().then(developers => {
             setDevelopers(developers);
+        }).catch(() => {
+            setDevelopers([
+                {
+                    "name": "The developer",
+                    "avatar_image": "https://upload.wikimedia.org/wikipedia/commons/thumb/5/52/LinuxCon_Europe_Linus_Torvalds_03.jpg/200px-LinuxCon_Europe_Linus_Torvalds_03.jpg",
+                    "description": "This guy really know about development. He is a icon of tech.",
```
Comment (lucasmauro) [inline src/pages/Developers.tsx]: This guy really know**s** about development. He is a**n** icon of tech.

Comment (lucasmauro) [reply src/pages/Developers.tsx]: This is great, by the way. Linus Torvalds... LOL

**Review by mspdl — APPROVED** [review]
Looks good 🥇

**Pull request merged**